src/toil/lib/aws/utils.py

```diff
@@ -232,9 +301,9 @@ def get_object_for_url(url: ParseResult, existing: Optional[bool] = None) -> Obj
         """
         s3_resource = cast(S3ServiceResource, session.resource('s3'))
```

Comment:
This looks like it's unused since there is `s3` below?

Reply:
Hm. We probably don't need this.
```suggestion
```